.evergreen/config.yml

```diff
@@ -119,3 +129,15 @@ buildvariants:
       - rhel87-small
     tasks:
       - name: test-langchain-python
+
+  - name: test-chatgpt-retrieval-plugin-rhel
+    display_name: ChatGPT Retrieval Plugin
+    expansions:
+      DIR: chatgpt-retrieval-plugin
+      REPO_NAME: chatgpt-retrieval-plugin
+      CLONE_URL: -b feature/mongodb-datastore --single-branch https://github.com/caseyclements/chatgpt-retrieval-plugin.git
```

Comment:
We should mark a TODO for this section as well. Once the chatgpt-retrieval-plugin gets merged, we'll want to clone their main/master branch.

Reply:
That's a good idea. I'll do that now. I'll also reference it from the jira.